backend/src/Validation.hs

```diff
@@ -110,10 +110,10 @@ validateVictory report
     fpmv = report.side == Free && report.victory == Military
     victory = (report.side, report.victory)
     correctVictory
-      | report.corruption >= 12 = Just (Shadow, Ring)
-      | report.mordor == Just 5 = Just (Free, Ring)
       | report.side == Free && report.victory == Concession = Just (Free, Concession)
```

Comment:
Turns out to be a much softer world if we just consider any concession as authoritative. And I think I'm okay with that compromise. You?